<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beamline Status</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="beamline_box">
     <property name="title">
      <string>Beamline</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="beamline_layout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="SR_current_label">
          <property name="text">
           <string>Stored Beam:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="PyDMLabel" name="SR_current_RBV_label">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>102.1 mA</string>
          </property>
          <property name="channel" stdset="0">
           <string>ca://S:SRcurrentAI</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="SR_mode_label">
          <property name="text">
           <string>Mode:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="PyDMLabel" name="SR_mode_RBV_label">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Top-Up</string>
          </property>
          <property name="channel" stdset="0">
           <string>ca://S:ActualMode</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="shutter_permit_label">
          <property name="text">
           <string>Shutter Permit:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="SCDS_label">
          <property name="text">
           <string>Shutter C/D:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="energy_label">
          <property name="text">
           <string>Energy:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="PyDMLabel" name="energy_RBV_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>4000 eV</string>
          </property>
          <property name="precision" stdset="0">
           <number>2</number>
          </property>
          <property name="showUnits" stdset="0">
           <bool>true</bool>
          </property>
          <property name="precisionFromPV" stdset="0">
           <bool>false</bool>
          </property>
          <property name="channel" stdset="0">
           <string>haven://monochromator_energy_user_readback</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="PyDMByteIndicator" name="shutter_permit_indicator">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="alarmSensitiveContent" stdset="0">
           <bool>false</bool>
          </property>
          <property name="alarmSensitiveBorder" stdset="0">
           <bool>true</bool>
          </property>
          <property name="channel" stdset="0">
           <string>ca://ACIS:ShutterPermit</string>
          </property>
          <property name="onColor" stdset="0">
           <color>
            <red>212</red>
            <green>237</green>
            <blue>218</blue>
           </color>
          </property>
          <property name="offColor" stdset="0">
           <color>
            <red>235</red>
            <green>241</green>
            <blue>181</blue>
           </color>
          </property>
          <property name="showLabels" stdset="0">
           <bool>true</bool>
          </property>
          <property name="bigEndian" stdset="0">
           <bool>false</bool>
          </property>
          <property name="circles" stdset="0">
           <bool>false</bool>
          </property>
          <property name="labelPosition" stdset="0">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="numBits" stdset="0">
           <number>1</number>
          </property>
          <property name="shift" stdset="0">
           <number>0</number>
          </property>
          <property name="labels" stdset="0">
           <stringlist>
            <string></string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="PyDMLabel" name="mono_mode_RBV_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Fixed offset</string>
          </property>
          <property name="channel" stdset="0">
           <string>haven://monochromator_mode</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="mono_mode_label">
          <property name="text">
           <string>Mono Mode:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="FES_label">
          <property name="text">
           <string>Shutter A:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="shutter_A_layout">
          <item>
           <widget class="PyDMByteIndicator" name="FES_state_indicator">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="alarmSensitiveContent" stdset="0">
             <bool>false</bool>
            </property>
            <property name="alarmSensitiveBorder" stdset="0">
             <bool>true</bool>
            </property>
            <property name="channel" stdset="0">
             <string>haven://front_end_shutter_pss_state</string>
            </property>
            <property name="showLabels" stdset="0">
             <bool>true</bool>
            </property>
            <property name="bigEndian" stdset="0">
             <bool>false</bool>
            </property>
            <property name="circles" stdset="0">
             <bool>false</bool>
            </property>
            <property name="labelPosition" stdset="0">
             <enum>QTabWidget::South</enum>
            </property>
            <property name="numBits" stdset="0">
             <number>1</number>
            </property>
            <property name="shift" stdset="0">
             <number>0</number>
            </property>
            <property name="labels" stdset="0">
             <stringlist>
              <string></string>
             </stringlist>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PyDMPushButton" name="FES_open_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="channel" stdset="0">
             <string>haven://front_end_shutter_open_signal</string>
            </property>
            <property name="pressValue" stdset="0">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="shutter_CD_layout">
          <item>
           <widget class="PyDMByteIndicator" name="SCDS_state_indicator">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="alarmSensitiveContent" stdset="0">
             <bool>false</bool>
            </property>
            <property name="alarmSensitiveBorder" stdset="0">
             <bool>true</bool>
            </property>
            <property name="channel" stdset="0">
             <string>haven://hutch_shutter_pss_state</string>
            </property>
            <property name="showLabels" stdset="0">
             <bool>true</bool>
            </property>
            <property name="bigEndian" stdset="0">
             <bool>false</bool>
            </property>
            <property name="circles" stdset="0">
             <bool>false</bool>
            </property>
            <property name="labelPosition" stdset="0">
             <enum>QTabWidget::South</enum>
            </property>
            <property name="numBits" stdset="0">
             <number>1</number>
            </property>
            <property name="shift" stdset="0">
             <number>0</number>
            </property>
            <property name="labels" stdset="0">
             <stringlist>
              <string></string>
             </stringlist>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PyDMPushButton" name="SCDS_open_button">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
            <property name="channel" stdset="0">
             <string>haven://hutch_shutter_open_signal</string>
            </property>
            <property name="pressValue" stdset="0">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PyDMPushButton" name="SCDS_close_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
            <property name="channel" stdset="0">
             <string>haven://hutch_shutter_close_signal</string>
            </property>
            <property name="pressValue" stdset="0">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="bss_box">
     <property name="title">
      <string>Proposal/ESAF</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="bss_layout">
        <property name="rowWrapPolicy">
         <enum>QFormLayout::DontWrapRows</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Proposal:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="proposal_layout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="proposal_id_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>#######</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="proposal_title_label">
            <property name="text">
             <string>&lt;Unset&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ESAF:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="esaf_layout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="esaf_id_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>#######</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="esaf_title_label">
            <property name="text">
             <string>&lt;Unset&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>End Date:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Users:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="esaf_end_date_label">
          <property name="text">
           <string>&lt;Unset&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="esaf_users_label">
          <property name="text">
           <string>&lt;Unset&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="esaf_status_label">
          <property name="text">
           <string>&lt;Unset&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bss_modify_button">
          <property name="toolTip">
           <string>Load the window for changing beamline scheduling system (BSS) details.</string>
          </property>
          <property name="text">
           <string>BSS</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMByteIndicator</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.byte</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
